lib/src/main/java/uk/gov/di/ipv/core/library/helpers/AuthorizationRequestHelper.java

```diff
@@ -55,9 +57,16 @@ public static SignedJWT createSignedJWT(
 
         String criId = credentialIssuerConfig.getId();
 
-        URI redirectionURI =
-                getRedirectionURI(
-                        criId, configurationService.getSsmParameter(CORE_FRONT_CALLBACK_URL));
+        URI redirectionURI;
+        if (DCMAW_CRI_IDS.contains(criId)) {
```

Comment:
After looking back on this part, I might change this up a bit. It might be nicer to store the full redirect URI value in SSM and just use that rather than construct it here.

Reply:
I've updated the PR to do this. Plus this new PR to add the new params to ssm - https://github.com/alphagov/di-ipv-config/pull/876/files#diff-07674faffb5f12730b95c927da7ae98beda7fad03267639510f0d2eadc56e7e2